### DISCUSSION TITLE
[feature request] : Different "quick actions" panels by profiles.

### DISCUSSION THREAD

**Discussion by MatthieuQ** [osmandapp/OsmAnd#11783] [Ideas]
Hello, 

It would be great to have multiple "quick actions" panels based on profiles. 

For example, with my "open street map" profile I don't need the same "quick actions" as in my "navigation" profile.

Don't you think it could be great ?

**Comment by vshcherb** [comment]
Yes, it was suggested before though simple solution would be to have separator and then save the quick action panel state on each profile
